templates/typescript-app/cdk8s.yaml

```diff
@@ -1,4 +1,4 @@
 language: typescript
-app: ts-node main.ts
```

Comment:
How about we just do `npx ts-node main.ts`? Normally our direction of reasoning is `package.json -> cdk8s.yaml`, not the other way around :)

Reply:
@shinebayar-g `npx` is included with the node distribution, so its not an additional dependency. This is also the way we do it in the aws-cdk: https://github.com/aws/aws-cdk/blob/main/packages/aws-cdk/lib/init-templates/app/typescript/cdk.template.json#L2

`npx` will also utilize the local `node_modules` dependencies and has the benefit of installing it on-demand if its missing. I'd like to keep the `cdk8s.yaml` file self contained and clean of `package.json` scripts.

Reply:
> npx will also utilize the local node_modules dependencies and has the benefit of installing it on-demand if its missing.

Oh I completely missed this detail. Just checked the behavior, you're right. I always thought it installs its own dependency with own execution environment, that was false. It prefers local first.

I'll update the PR.